vue-frontend/src/pages/jobs/apply/[id].vue

```diff
@@ -366,28 +361,38 @@ export default {
     }),
   },
   methods: {
+    async setJob() {
+      if (this.job == null || this.job.id != this.id) {
+        await this.getCareerDetails();
+      }
+      this.setCareerDetails();
+    },
     async getCareerDetails() {
       var req = {
         jobId: this.id,
         href: this.$route.href,
       };
-      await store.dispatch("getJobsById", req);
-      if (this.jobsError == null) {
```

Comment:
Shouldn't we have to check `jobsError`?

Reply:
No, in case of error it will go to catch block.